src/createViewer.js

```diff
@@ -702,8 +702,8 @@ const createViewer = async (
   }
 
   publicAPI.setImage = async (image, name) => {
-    if (typeof name === 'undefined' && context.images.selectedName) {
-      name = context.images.selectedName
+    if (typeof name === 'undefined') {
+      name = context.images?.selectedName || 'Image'
```

Comment:
Cool!  What do you think of the null coalesceing thing?
```
const foundName =  name ?? context.images?.selectedName ?? 'Image'
...
multiscaleImage.name = foundName
```

Reply:
I love a nice one-liner, thanks for the suggestion!